fly.toml

```diff
@@ -0,0 +1,43 @@
+# fly.toml app configuration file generated for nijikai-go on 2025-03-19T13:34:15+09:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = 'nijikai-go'
+primary_region = 'nrt'
+console_command = '/rails/bin/rails console'
+
+[build]
+
+[deploy]
+  release_command = "./bin/rails db:prepare"
+
+[http_service]
+  internal_port = 3000
+  force_https = true
+  auto_stop_machines = 'stop'
+  auto_start_machines = true
```

Comment:
- [App configuration (fly.toml) - The http_service section](https://fly.io/docs/reference/configuration/#the-http_service-section)
  - https://fly.io/docs/reference/configuration/#http_service-concurrency
    - [Guidelines for concurrency settings · Fly Docs](https://fly.io/docs/apps/concurrency/)
- [Fly Proxy autostop/autostart · Fly Docs](https://fly.io/docs/reference/fly-proxy-autostop-autostart/)
- [Autostop/autostart Machines · Fly Docs](https://fly.io/docs/launch/autostop-autostart/)